Library/Homebrew/cmd/pull.rb

```diff
@@ -179,9 +193,9 @@ def pull
             package = Bintray.package f.name
             version = f.pkg_version
             curl "-w", '\n', "--silent", "--fail",
-              "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
-              "-d", '{"publish_wait_for_secs": -1}',
-              "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
+                 "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
+                 "-d", '{"publish_wait_for_secs": -1}',
+                 "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
```

Comment:
Same here. Sorry if I'm being somewhat picky, but you haven't changed the code, just reformatted it. That's not good for reviewing and for a clean history, so I'd prefer if this was reverted.